tasks.py

```diff
@@ -42,3 +42,26 @@ def imageid(ctx):
         image_name=DOCKER_IMAGE_NAME,
         format='{{.Id}}'
     ))
+
+def link_phutil_map(ctx):
+    ctx.run("ln -fs `pwd`/phutil_map /tmp/phutil_map")
+
+@task
+def build_test(ctx):
+    """Test phabricator extensions."""
+    link_phutil_map(ctx)
+    ctx.run("docker-compose -f docker-compose.test.yml build phabricator")
```

Comment:
I can't find any mention of circle supporting docker-compose in the 1.0 product. I only see support for it in the 2.0 beta.